examples/user_property.rs

```diff
@@ -5,16 +5,36 @@ use onig::*;
 
 fn main() {
 
-    define_user_property("HandakuonHiragana",
-                         &[(0x3071, 0x3071), // PA
-                           (0x3074, 0x3074), // PI
-                           (0x3077, 0x3077), // PU
-                           (0x307a, 0x307a), // PE
-                           (0x307d, 0x307d)]); // PO
+    define_user_property(
+        "HandakuonHiragana",
+        &[
+            (0x3071, 0x3071), // PA
+            (0x3074, 0x3074), // PI
+            (0x3077, 0x3077), // PU
+            (0x307a, 0x307a), // PE
+            (0x307d, 0x307d),
+        ],
+    ); // PO
```

Comment:
I think this comment should be moved to line 15.

Reply:
Nice catch, that should be fixed now